### PR TITLE
User supplied room ids

### PR DESCRIPTION
**Pull request by vivangkumar**
### What?

Allows supplying room id when creating a room.

### Why?

----

### CODE REVIEW

File: Sources/ChatManager.swift [hunk 1]
```diff
@@ -50,7 +50,7 @@ import NotificationCenter
 
         self.logger = logger
 
-        let sdkInfo = PPSDKInfo(productName: "chatkit", sdkVersion: "1.6.0")
+        let sdkInfo = PPSDKInfo(productName: "chatkit", sdkVersion: "1.7.0")
```
Comment (callum-oakley) [inline Sources/ChatManager.swift]: sucks that you have to manually update this every time. I wonder if you can pull it out of the podspec? (out of scope for this PR, but would be cool if it isn't a lot of work)

File: Sources/ChatManager.swift [hunk 2]
```diff
@@ -166,7 +166,7 @@ import NotificationCenter
                         id: strongSelf.userID,
                         pathFriendlyID: strongSelf.pathFriendlyUserID,
                         instance: strongSelf.instance,
-                        v5Instance: strongSelf.v5Instance,
+                        v5Instance: strongSelf.v6Instance,
```
Comment (callum-oakley) [inline Sources/ChatManager.swift]: `v5instances`

File: Sources/PCCurrentUser.swift [hunk 3]
```diff
@@ -81,7 +81,7 @@ public final class PCCurrentUser {
         self.avatarURL = avatarURL
         self.customData = customData
         self.instance = instance
-        self.v5Instance = v5Instance
+        self.v6Instance = v5Instance
```
Comment (callum-oakley) [inline Sources/PCCurrentUser.swift]: more v5

Comment (vivangkumar) [reply Sources/PCCurrentUser.swift]: Stupid Xcode..I used the refactor thing where it should change everything..but I don't think it quite worked..

**Comment by vivangkumar** [comment]
Ok I think that should be it.

**Pull request merged**